x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/utils.ts

```diff
@@ -1027,3 +1052,92 @@ export const getDisabledActionsWarningText = ({
     return `${alertsGeneratedText} connector ${actionTypesJoined} is not enabled. To send notifications, you need a higher Security Analytics license / tier`;
   }
 };
+
+export const isEqlBuildingBlockAlert = (
+  alertObject: BaseFieldsLatest
+): alertObject is EqlBuildingBlockFieldsLatest =>
+  (alertObject as EqlBuildingBlockFieldsLatest)?.[ALERT_BUILDING_BLOCK_TYPE] != null;
+
+export const isEqlShellAlert = (
+  alertObject: BaseFieldsLatest
+): alertObject is EqlShellFieldsLatest =>
+  (alertObject as EqlShellFieldsLatest)?.[ALERT_UUID] != null &&
+  (alertObject as EqlShellFieldsLatest)?.[ALERT_GROUP_ID] != null;
+
+export type RuleWithInMemorySuppression =
+  | ThreatRuleParams
+  | EqlRuleParams
+  | MachineLearningRuleParams;
```

Comment:
This type is a duplicate of another with the same name, let's unify them

Reply:
Fixed here: https://github.com/elastic/kibana/pull/189725/commits/479b16a24e7e16bfdea473b6a80c32d44dd6ff7b#diff-1db264da3d27dbbc4e709f64e37298dd7c92635baa716f1d71c86d78f7c8d27fL31-L32